app/src/marketplace/containers/EditProductPage/publishQueue.js

```diff
@@ -14,6 +14,7 @@ export const actionsTypes = {
     PUBLISH_FREE: 'publishFree',
     UNPUBLISH_FREE: 'unpublishFree',
     PUBLISH_PENDING_CHANGES: 'publishPendingChanges',
+    SET_REQUIRES_WHITELIST: 'setRequiresWhitelist',
```

Comment:
This new transaction should be added to the publish modal as well (translation string for the progress bar).